styles/styles.css

```diff
@@ -0,0 +1,169 @@
+* {
+    box-sizing: border-box;
+}
+
+a {
+    text-decoration: none;
+}
+
+.d-flex {
+    display: flex;
+}
+
+.d-block {
+    display: block;
+}
+
+.full-width{
+    width: 100%;
+}
+
+body {
+    width: 100%;
+    max-width: 1080px;
+    margin: 0 auto;
+    font-family: 'Montserrat', sans-serif;
+    background-color: #ebebeb;
+    color: black;
+}
+
+.menu {
+/*
+    
+*/
+    width: 100%;
+    max-width: 1080px;
+    z-index: 1;
+    position: fixed;
+    background-color: #373c3f;
+    flex-wrap: wrap;
+}
+
+.small-screen-menu {
+    font-weight: 500;
+}
+
+.small-screen-menu span {
+    width: 87%;
+    padding-left: 7%;
+    padding-bottom: 3.7%;
+    padding-top: 3.7%;
+    align-items: center;
+    color: white;
+}
+
+.small-screen-menu .menu-button {
+    width: 13%;
+    align-items: center;
+    justify-content: center;
+    background-color: black;
+    color: white;
+}
+
+.big-screen-menu {
+    padding-top: 15px;
+    padding-bottom: 15px;
+    font-weight: 700;
+}
+
+.big-screen-menu ul {
+    list-style: none;
+}
+
+.big-screen-menu a {
+    padding-left: 10px;
+    padding-right: 10px;
+    align-items: center;
+    justify-content: center;
+    color: white;
+}
+
+.big-screen-menu a:hover {
+    color: #a0a0a0;
+}
+
+.banner {
+    position: relative;
+    background: linear-gradient(to bottom right, #23c4d7, #078aa4);
+}
+
+.banner-content {
+    position: absolute;
+    left: 10%;
+    top: 50%;
+    color: white;
+}
+
+.banner-content .description {
+    font-weight: bold;
+}
+
+.banner-content .link {
+    margin-top: 3%;
+}
+
+.banner-content .link span {
+    width: 90%;
+    color: #a6d5df;
+}
+
+.banner-content .link i {
+    width: 10%;
+    color: white;
+    text-align: right;
+}
+
+.content1 {
+    background-color: #dc53d0;
+}
+
+.content1 .description{
+    width: 80%;
+    padding: 3em;
+    background-color: #27212c;
+    color: #82738f;
+}
+
+.content1 a {
+    width: 20%;
+    margin-top: 1px;
+    align-items: center;
+    justify-content: center;
+    background-color: #bc3183;
+}
+
+.content1 i {
+    text-align: center;
+}
+
+.gallery {
+    flex-wrap: wrap;
+}
+
+.gallery a {
+    align-items: center;
+    justify-content: center;
+    background-color: #5b0fa1;
+}
+
+.gallery i {
+    color: white;
+}
+
+main article {
```

Comment:
I'd highly recommend you to use classes instead of tags in case of browser performance improvement and styles compatibility issues with other libs